pygenesys/data/eia_data.py

```diff
@@ -63,6 +66,10 @@ def get_eia_generators(month=None, year=None):
         'County'
     ]
 
+    # initialize 
+    m = 'thermidor'
+    y = 2
+
```

Comment:
Is there any reason you need to initialize these variables, but not the `d` variable? Is `d` used anywhere?

Reply:
d is not used because this dataset is only updated monthly. `get_date` returns a month, day, and year for generality.